arex-instrumentation/redis/arex-redission-v3/src/main/java/io/arex/inst/redisson/v3/RedissonWrapperCommon.java

```diff
@@ -20,6 +20,9 @@ public static <R> RFuture<R> delegateCall(String redisUri, String cmd, String ke
 
     public static <R> RFuture<R> delegateCall(String redisUri, String cmd, String key, String field,
                                               Callable<RFuture<R>> futureCallable) {
+        if (ContextManager.needRecord()) {
```

Comment:
Spring Data Redis integrates with Lettuce and Jedis, why was Redisson modified?

Reply:
RedisTemplate calls RedisConnectionFactory to create a RedisConnection object. The implementation classes of the RedisConnection interface include JedisConnection, LettuceConnection provided by spring-data-redis and RedissonConnection provided by redisson-spring-data.

Reply:
RedisTemplate calls RedisConnectionFactory to create a RedisConnection object. The implementation classes of the RedisConnection interface include JedisConnection, LettuceConnection provided by spring-data-redis and RedissonConnection provided by redisson-spring-data.